webapp/src/main/java/org/iqu/webapp/news/management/RetrieveCategoriesService.java

```diff
@@ -19,22 +23,19 @@
 @Path("/categories")
 public class RetrieveCategoriesService {
 
```

Comment:
all the calls to news-related methods should be under the same endpoint (could be named NewsEndpoint-> /news). These include news categories (/news/categories), news authors (/news/authors) and so on. The same for events and sources and other endpoints. So basically there should be a NewsEndpoint service under which getNews, getCategories, getAuthors should be handled.